src/components/People.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from "react";
+
+
+
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state = {
+            people: [],
+            userInput: ""
+        };
+    };
+
+    fetchPeople = () => {
+        fetch("https://ghibliapi.herokuapp.com/people")
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                people: data,
+
+            });
+        });
+    };
+
+    componentDidMount = () => {
+        this.fetchPeople();
+    };
+
+
+    handleOnChange = (event) => {
+        this.setState({
+            userInput: event.target.value
+        })
+
+    }
+
+    handleOnSubmit = (e) => {
+        e.preventDefault();
+        
+
+    }
```

Comment:
```suggestion
    handleOnSubmit = (e) => {
        e.preventDefault();
        let characterMatch = this.state.people.find((person) => {
                return person.name === this.state.userInput
        })
        this.setState({
          selectedCharacter: characterMatch
        })
    }
```
Now that you're saving the `selectedCharacter` in state, you can update who the current `selectedCharacter` each time the user searches for a name. 
If `.find()`is able to find a match, `selectedCharacter` will be set to an object that represents the matching character. If `.find()` is NOT able to find a match, `selectedCharacter` will be set to _`undefined`_.